src/os2datascanner/projects/admin/adminapp/templates/miniscan.html

```diff
@@ -56,6 +68,14 @@ <h2>{% trans "File to scan" %}</h2>
               {% trans "Please, select another (smaller) file to scan." %}
             </div>
           {% endwith %}
+
+          <h2>{% trans "Enter your text below" %}</h2>
+          <input name="text"
```

Comment:
This should be a `<textarea>` and not an `<input>`, I think.

Reply:
i don't really see the difference it would make. For me, they both recieve text, and <textarea> is just bigger.

Reply:
That's sort of the point. :-) The text box I'm typing this in, for example, is a `<textarea>` precisely because it's nicer to type longer texts into the larger box (and to be able to insert new lines).

Reply:
i didn't think about the newline stuff. I'll change it to <textarea>.